### PR TITLE
Update links to reach docs.prestashop-project.org instead of doc.prestashop.com

### LINKED ISSUE #27493: Use docs.prestashop-project.org links instead of doc.prestashop.com
Use docs.prestashop-project.org links instead of doc.prestashop.com

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Update links to docs used by the cards into Back Office.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Fixed ticket?     | Fix https://github.com/PrestaShop/PrestaShop/issues/27493
| Deprecations?     | no
| How to test?      | See that helper cards have the new links used. All helper card redirect to prestashop-project.org instead of doc.prestashop.com
| Possible impacts? | n/d.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27150)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]
nice ! thank you

**Review by PierreRambaud — DISMISSED** [review]
I don't think it should target the 1.7.8.x branch.
Could you please target the develop branch?

**Comment by PrestaEdit** [comment]
I could, but it's specific to 1.7 version here. We will need to update again the links for develop branch.

**Review by sowbiba — DISMISSED** [review]

**Comment by PrestaEdit** [comment]
Related to https://github.com/PrestaShop/PrestaShop/issues/24651

**Comment by PierreRambaud** [comment]
> I could, but it's specific to 1.7 version here. We will need to update again the links for develop branch.

It's legit, but it hasn't been planned to integrate this in 1.7.8.4. Ping @MatShir

**Comment by MatShir** [comment]
Nice catch, it would be nice to have it for the patch.

**Review by MatShir — CHANGES_REQUESTED** [review]
@Julievrz ping for a double check :)

**Review by MatShir — APPROVED** [review]

**Review by MatShir — CHANGES_REQUESTED** [review]
@Julievrz ping for a double check :)

**Review by PierreRambaud — DISMISSED** [review]
The boss approves, I have to follow him!

File: src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml [hunk 1]
```diff
@@ -8,78 +8,80 @@ services:
...
-          'es': 'https://doc.prestashop.com/display/PS17/Paginas+-+Gestionar+el+Contenido+Estatico'
-          'it': 'https://doc.prestashop.com/display/PS17/Pagine+-+Gestione+Contenuti+Statici'
-          '_fallback': 'https://doc.prestashop.com/display/PS17/Pages+-+Managing+Static+Content'
+          'en': 'https://docs.prestashop-project.org/1.7-documentation/user-guide/improving-shop/customizing-store-design/pages-managing-static-content'
+          'fr': 'https://docs.prestashop-project.org/1.7-documentation/v/french/guide-utilisateur/optimiser-boutique/personnaliser-apparence-boutique/pages-gerer-contenu-statique'
+          'es': 'https://docs.prestashop-project.org/1.7-documentation/v/spanish/guia-usuario/optimizar-tienda/personalizar-diseno-tienda/paginas-gestionar-contenido-estatico'
+          'it': 'https://docs.prestashop-project.org/1.7-documentation/v/italian/guida-utente/migliorare-negozio/personalizzare-design-negozio/pagine-gestione-contenuti-statici'
+          '_fallback': 'hhttps://docs.prestashop-project.org/1.7-documentation/user-guide/improving-shop/customizing-store-design/pages-managing-static-content'
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml]:
```suggestion
          '_fallback': 'https://docs.prestashop-project.org/1.7-documentation/user-guide/improving-shop/customizing-store-design/pages-managing-static-content'
```

Comment (PrestaEdit) [reply src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml]: Nice catch.

File: src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml [hunk 2]
```diff
@@ -8,78 +8,80 @@ services:
...
-          'fr': 'http://doc.prestashop.com/pages/viewpage.action?pageId=51840325#Param%%C3%%A8tresdeperformances-Modedebug'
-          'it': 'http://doc.prestashop.com/pages/viewpage.action?pageId=54266514'
-          'es': 'https://doc.prestashop.com/display/PS17/Rendimiento#Rendimiento-ModoDepuraci%%C3%%B3n'
-          '_fallback': 'https://doc.prestashop.com/display/PS17/Performance#Performance-Debugmode'
+          'en': 'https://docs.prestashop-project.org/1.7-documentation/user-guide/configuring-shop/advanced-parameters/performance#performance-debugmode'
+          'fr': 'https://docs.prestashop-project.org/1.7-documentation/v/french/guide-utilisateur/configurer-boutique/parametres-avances/performance#parametresdeperformances-modedebug'
+          'es': 'https://docs.prestashop-project.org/1.7-documentation/v/spanish/guia-usuario/configurar-tienda/parametros-avanzados/rendimiento#rendimiento-mododepuracion'
+          'it': 'https://docs.prestashop-project.org/1.7-documentation/v/italian/guida-utente/configurare-negozio/parametri-avanzati/performance-prestazioni#performance-prestazioni-modalitadidebug'
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml]:
```suggestion
          'it': 'https://docs.prestashop-project.org/1.7-documentation/v/italian/guida-utente/configurare-negozio/parametri-avanzati/performance-prestazioni#modalita-di-debug'
```

File: src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml [hunk 3]
```diff
@@ -8,78 +8,80 @@ services:
...
+          'it': 'https://docs.prestashop-project.org/1.7-documentation/v/italian/guida-utente/vendere/gestire-ordini/note-di-credito'
+          '_fallback': 'https://docs.prestashop-project.org/1.7-documentation/user-guide/selling/managing-orders/credit-slips'
         'carrier':
-          'en': 'https://doc.prestashop.com/display/PS17/Carriers'
-          'fr': 'https://doc.prestashop.com/display/PS17/Gerer+les+transporteurs'
-          'es': 'http://doc.prestashop.com/display/PS17/Gestionar+Transportistas'
-          'it': 'http://doc.prestashop.com/display/PS17/Corrieri'
-          '_fallback': 'https://doc.prestashop.com/display/PS17/Carriers'
+          'en': 'https://docs.prestashop-project.org/1.7-documentation/user-guide/improving-shop/managing-shipping/carriers'
+          'fr': 'https://docs.prestashop-project.org/1.7-documentation/v/french/guide-utilisateur/optimiser-boutique/gerer-livraisons'
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml]:
```suggestion
          'fr': 'https://docs.prestashop-project.org/1.7-documentation/v/french/guide-utilisateur/optimiser-boutique/gerer-livraisons/gerer-transporteurs'
```

**Comment by Julievrz** [comment]
> @Julievrz ping for a double check :)

Done ✅

**Review by MatShir — DISMISSED** [review]

**Comment by florine2623** [comment]
Hello @PrestaEdit ,

Could you create an issue  and link it in the PR please ?
Thanks!

**Comment by PrestaEdit** [comment]
Hi,

There is really a need of ?

Also, I can see that this issue was mentionned by my own (https://github.com/PrestaShop/PrestaShop/issues/24651#ref-pullrequest-1091092616) but don't know when and how it's desapear from here...

EDIT: I remember, I noticed that it was not the same issue.

**Comment by florine2623** [comment]
@PrestaEdit 

As discussed without with @Robin-Fischer-PS , an issue is needed. PM should go over the issue and approve it before we can test it ^^

Thanks!

**Comment by MatShir** [comment]
Thanks @florine2623 for following the process, but for this PR an issue is not required here. The PR has been checked and approved 2 time by the product. All the information needed seems to be available on the PR description. Unless there is missing information, I think we can test it ;)

**Comment by florine2623** [comment]
Thanks for the details @MatShir

@PrestaEdit , thanks for the PR.
After some testing, all the links for "Full documentation" in Helper cards redirect to [doc.prestashop.com](https://doc.prestashop.com/)  :
Screenshot 2022-01-27 at 17 44 28

- In Dashboard
- In Orders > Shopping cart
- In Catalog > Attributes & Features > tab Attributes & In Catalog > Attributes & Features > tab Features
- In Catalog > Discounts > tab Cart Rules & In Catalog > Discounts > tab Catalog Price Rules
- In Customer Service > Customer Service
- In Customer Service > Merchandise Return
- Stats
- Design > Theme & Logo > Pages Configuration & Design > Theme & Logo > Advanced Parameters
- Design > Image Settings
- Shipping > Carriers
- International > Locations > Countries
- International > Taxes > Tax Rules
- Shop Parameters > Order Settings > Statuses
- Shop Parameters > Customer Settings > Groups
- Shop Parameters > Customer Settings > Titles
- Shop Parameters > Contact > Stores
- Shop Parameters > Traffic & SEO > Search Engines
- Shop Parameters > Traffic & SEO > Referrers
- Shop Parameters > Shop > Search & Shop Parameters > Shop > Tags
- Shop Parameters > Merchant Expertise
- Advanced Parameters > Team > Permissions
- Advanced Parameters > Multistore


Also, in search result, there is that :
Screenshot 2022-01-27 at 18 06 45
It should be updated with the new link

Could you take a look ?
Thanks!

**Comment by PrestaEdit** [comment]
It's not the goal of this PR, it's about links into Helper Card on some pages.

**Comment by PrestaEdit** [comment]
Like this one.

Capture d’écran 2022-01-27 à 18 23 31

**Comment by PrestaEdit** [comment]
I've try to add more replace  but there are 5 results that can't be changed, as it was 1.6 version and not available on the doc.

Capture d’écran 2022-01-27 à 20 50 07
4

**Comment by MatShir** [comment]
Oh nice! thanks for the extra work :)
For the rest, It looks like there is no doc as you mentionned it. I will created a dedicated issue. So with @Julievrz, we will see if we need to migrate the doc or use maybe the dev doc for some of it.
We will review your work on thuesday for a double check :)

**Comment by PrestaEdit** [comment]
Is the waiting for author still available ?

**Comment by Robin-Fischer-PS** [comment]
Hi,

I changed to Waiting for Dev because it needs review before getting back to QA.

**Review by Progi1984 — APPROVED** [review]

**Review by Julievrz — APPROVED** [review]
Thanks!

**Comment by florine2623** [comment]
Hello @PrestaEdit ,

LGTM, it is QA ✅ !

**Comment by Progi1984** [comment]
Thanks @PrestaEdit & @florine2623

**Pull request merged**